static-instrumentation/maven-plugin/build.gradle

```diff
@@ -0,0 +1,6 @@
+plugins {
+    id("otel.java-conventions")
+}
+
+ext.libraryName = 'OpenTelemetry Java Static Instrumentation Maven2 Plugin'
+description = 'Maven2 plugin for static instrumentation of projects code and dependencies'
```

Comment:
```suggestion
description = 'Maven plugin for static instrumentation of projects code and dependencies'
```

Reply:
After rethinking this - M2 has been EOLed 8 years ago. Let's just use v3 API for the plugin and state M3 compatibility only.